batch.py

```diff
@@ -774,17 +788,17 @@ def manage_concurrency_for_job(job: hb.job.Job):
 
             plink_output_prefix = gene_dict[gene]['plink']
             # prepare input files
-            prepare_input_job = batch.new_python_job(f'Prepare inputs for: {gene}')
-            manage_concurrency_for_job(prepare_input_job)
-            copy_common_env(prepare_input_job)
+            run_job = batch.new_python_job(f'Run association for: {gene}, {celltype}')
```

Comment:
Not a big deal, but if celltype is "more important", consider putting that first?